src/libstd/time/duration.rs

```diff
@@ -73,7 +73,7 @@ impl Duration {
     /// ```
     #[stable(feature = "duration", since = "1.3.0")]
     #[inline]
-    pub fn new(secs: u64, nanos: u32) -> Duration {
+    pub const fn new(secs: u64, nanos: u32) -> Duration {
```

Comment:
I don't think `checked_add` and `expect` are `const fn` yet and can't be made `const fn` yet

Reply:
Ugh, I'll remove `new` from the PR then.